slu-scripts/vpuf.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+const cryptoJs = require('crypto-js');
```

Comment:
Please move the script + the tests for it into a separate folder called `vpuf` since there will be several scripts inside the `slu-scripts` module

Reply:
done